### PR TITLE
First version of quality control pathogens page

### PR DESCRIPTION
**Pull request by EvaGarciaAlvarez**

### CODE REVIEW

**Comment by lauportell** [comment]
I think the page is nice and clear!
I don't have many comments. Maybe only for consistency, in NGS pathogens data and NGS analysis sections, it mentions "Considerations for sequencing data" and "Considerations for analysing sequencing data". Do we want to leave only "Considerations" to be aligned with the other secions and pages? I am okay with both approaches, it won't change much :)

**Review by lauportell — APPROVED** [review]

File: pathogen-characterisation/quality-control.md [hunk 1]
```diff
@@ -18,14 +16,125 @@ training:
...
+* Quality control of the preanalytical process
+    * Spiking wastewater samples with exogenous calibrated virus reference material, such as mengovirus or murine norovirus, before preanalytical processing in order to evaluate the efficiency of viral concentration and extraction procedures (by evaluating the recovery of the exogenous reference material introduced into the sample prior to preanalytical processing). 
+    * Including a negative control sample during the nucleic acid extraction procedure to evaluate for potential cross contamination.
+* Quality control of the quantitative analytical detection
+    * Standard calibrated reference material (such as EURM-019 / EURM-014 single stranded RNA provided by the Joint Research Center)
+
+In particular the European Commission Recommendation (EU) 2021/472 of 17 March 2021 has advocated the use of a common approach to establish a systematic surveillance of SARS-CoV-2 and its variants in wastewaters in the EU (OJ L 98 19.03.2021, p. 3, CELEX: Specific Quality Standards are indicated in this Recommendation document: standards for PCR/Digital-PCR (polymerase chain reaction), shortly described below:
```
Comment (bedroesb) [inline pathogen-characterisation/quality-control.md:86]: >EU (OJ L 98 19.03.2021, p. 3, CELEX: Specific Quality Standards are indicated in this Recommendation document: standards for PCR/Digital-PCR (polymerase chain reaction), shortly described below:

This sentence is missing a parenthesis and is difficult to read.

File: pathogen-characterisation/quality-control.md [hunk 2]
```diff
@@ -18,14 +16,125 @@ training:
...
+* (b) Additional normalisation controls using cross-assembly phage (c) or pepper mild mottle virus are recommended for this purpose.
+* (c) If data for none of viruses referred to in point (b) can be obtained, alternative parameters could be used provided they deliver equivalent corrections for meteorological or other influences causing fluctuations of the viral load, which are not related to the pandemic such as precipitation or other meteorological effects. 
 
-New content is announced on the [home page](/) and [news page](/about/news), so please check for updates there. You can also watch for changes on this page by using a free service like [Visual Ping](https://visualping.io/) or [Distill Web Monitor](https://distill.io/), or by using a [browser add-on](https://chrome.google.com/webstore/detail/distill-web-monitor/inlikjemeeknofckkjolnjbpehgadgge?hl=en).
 
+Further information on wastewater quality control can be found in Wastewater QC workflow in GalaxyTrakr (SSQuAWK3) (protocols.io) protocol that has a nice listing of quality indicators that were defined in the context of SARS-CoV-2, but are easily generalizable.
```
Comment (bedroesb) [inline pathogen-characterisation/quality-control.md:140]: "protocols.io" seems to miss a link?

File: pathogen-characterisation/quality-control.md (hunk 1 above)
Comment (EvaGarciaAlvarez) [reply pathogen-characterisation/quality-control.md]: Indeed, thanks! I've modified it

**Review by bedroesb — APPROVED** [review]

**Pull request merged**